src/Compilers/CSharp/Portable/CSharpResources.resx

```diff
@@ -7184,6 +7184,9 @@ To remove the warning, you can use /reference instead (set the Embed Interop Typ
   <data name="ERR_BadBinaryReadOnlySpanConcatenation" xml:space="preserve">
     <value>Operator '{0}' cannot be applied to operands of type '{1}' and '{2}' that are not UTF-8 byte representations</value>
   </data>
+  <data name="ERR_ImplicitlyTypedDefaultParameter" xml:space="preserve">
+    <value> Default not allowed for implicitly typed lambda parameter '{0}' </value>
```

Comment:
Extra leading and trailing spaces. Also, while we aren't perfectly consistent on this (see the above message...) we do generally try to end errors with a `.`. Finally, `Default` isn't really a good descriptor. Consider something more like `Implicitly typed lambda parameter '{0}' cannot have a default value.`